pkg/istio-agent/bootstrap_stream.go

```diff
@@ -58,11 +58,7 @@ func (b *bootstrapDiscoveryStream) Send(resp *discovery.DiscoveryResponse) error
 			sendToChannelWithoutBlock(b.errCh, fmt.Errorf("failed to update bootstrap from discovery: %v", err))
 			return nil
 		}
-		select {
-		case <-b.errCh:
-		default:
-			close(b.errCh)
-		}
+		close(b.errCh)
```

Comment:
I think this was trying to avoid double close -> panic. Not sure if that is possible though... as a followup this code may need more tests since it currently (not from this PR) has some issues in error handling